src/me/ryanhamshire/PopulationDensity/PopulationDensity.java

```diff
@@ -1234,6 +1234,16 @@ public void TeleportPlayerToRegion(Player player, RegionCoordinates region, int
 		teleportDestination.setX(teleportDestination.getBlockX() + 0.5D);
 		teleportDestination.setY(teleportDestination.getBlockY() + 1D);
 		teleportDestination.setZ(teleportDestination.getBlockZ() + 0.5D);
+		
+		// Check the world border
+		double size = border.getSize() / 2;
+		Location center = border.getCenter();
+		double x = teleportDestination.getBlockX() - center.getX(),
+			z = teleportDetination.getBlockZ() - center.getZ();
+		if((x > size || (-x) > size) || (z > size || (-z) > size)) {
```

Comment:
So, I believe there is a WorldBorder#isInside method https://hub.spigotmc.org/javadocs/spigot/org/bukkit/WorldBorder.html#isInside-org.bukkit.Location-

I've seen a couple of different ways to do the math so this is yet another way I've seen it lol

Reply:
You could also do absolute value of x instead of having to invert it. But seems good otherwise.